private/pkg/git/git_test.go

```diff
@@ -206,59 +206,59 @@ func createGitDirs(
 ) (string, string) {
 	tmpDir := t.TempDir()
 
-	submodulePath := filepath.Join(tmpDir, "submodule")
+	submodulePath := normalpath.Join(tmpDir, "submodule")
```

Comment:
You can't use a normal path with the `os` package.

Reply:
You're right. Changed to only use `normalpath.Unnormalize` before passing it as elements to Join, and `filepath` pkg for the rest of it.